db/schema.rb

```diff
@@ -10,11 +10,19 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170418181155) do
+ActiveRecord::Schema.define(version: 20170419114516) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "help_requests", force: :cascade do |t|
+    t.string   "title"
+    t.string   "message"
+    t.boolean  "status"
```

Comment:
Shouldn't this be `urgent`rather than `status`? `status` `true/false` does not make sense.